lte/gateway/c/session_manager/sessiond_main.cpp

```diff
@@ -206,6 +212,9 @@ int main(int argc, char* argv[]) {
 
   std::shared_ptr<magma::AsyncSpgwServiceClient> spgw_client;
   std::shared_ptr<aaa::AsyncAAAClient> aaa_client;
+
+  /*AMF service client to handle response message*/
```

Comment:
We can use `//` for single line comments. :)

Reply:
Converted

Reply:
Changed